app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/branding/components/Branding.tsx

```diff
@@ -54,20 +54,24 @@ export const Branding = ({ hasBrandingRequestForm }: { hasBrandingRequestForm: b
   const savedErrorMessage = t("settingsResponseDelivery.savedErrorMessage");
 
   const handleSave = useCallback(async () => {
-    try {
-      await updateTemplate({
-        id,
-        formConfig: JSON.parse(getSchema()),
-        name: getName(),
-      });
-    } catch (e) {
-      logMessage.error(e);
-      toast.error(savedErrorMessage);
+    const result = await updateTemplate({
+      id,
+      formConfig: JSON.parse(getSchema()),
```

Comment:
Thinking about this now. I wonder if the try-catch originally here should stay because of JSON.parse

Reply:
ya we need the catch --- tested